spec/rails_helper.rb

```diff
@@ -12,6 +12,7 @@
 require 'rspec/rails'
 require 'spec_helper'
 require 'email_spec'
+require 'email_spec/rspec'
```

Comment:
We think this won't affect overall test suite runtime, but let's revisit if we notice a slowdown.